exec/vector/src/main/codegen/templates/BaseReader.java

```diff
@@ -48,6 +49,14 @@
     int size();
     void copyAsValue(MapWriter writer);
   }
+
+  public interface DictReader extends RepeatedMapReader {
```

Comment:
Please provide javadoc for interface and methods. This is really very interesting to know difference between 
```java
    int find(String key);
    int find(int key);
```
and when to use them.